cases/aeria/index.ts

```diff
@@ -0,0 +1,110 @@
+import { silentValidator } from '@aeriajs/validation';
+import { createCase } from '../../benchmarks';
+
+import {} from './index.d';
```

Comment:
```suggestion
import './index.d';
```

Should be sufficient, no?

Reply:
https://github.com/moltar/typescript-runtime-type-benchmarks/pull/1347#discussion_r1825191962